.circleci/config.yml

```diff
@@ -241,3 +241,7 @@ jobs:
       - run:
           name: Report test coverage results to CodeClimate
           command: ./cc-test-reporter after-build --coverage-input-type simplecov --exit-code $?
+      - store_artifacts:
+          path: tmp/capybara
+      - store_artifacts:
+          path: log/test.log
```

Comment:
Stuff capybara screenshots and the Rails log in CI so we can take a look when tests fail.